src/SVGPathDataTransformer.ts

```diff
@@ -149,6 +154,52 @@ function NORMALIZE_HVZ(
       delete command.lArcFlag;
       delete command.sweepFlag;
     }
+
+    // Check for quad curves that are lines
+    if (normalizeC && command.type & SVGPathData.QUAD_TO) {
```

Comment:
i'm not sure if this should not be separate command

Reply:
Seems fine to me, seems to be about the same class of transform as the curve conversion.